source/plugins/go/src/oms.go

```diff
@@ -1736,6 +1762,9 @@ func InitializePlugin(pluginConfPath string, agentVersion string) {
 	Log("kubeMonAgentConfigEventFlushInterval = %d \n", kubeMonAgentConfigEventFlushInterval)
 	KubeMonAgentConfigEventsSendTicker = time.NewTicker(time.Minute * time.Duration(kubeMonAgentConfigEventFlushInterval))
 
+	Log("containerLogV2PollingIntervalSeconds = %d \n", containerLogV2PollingIntervalSeconds)
```

Comment:
Change the logging statement...Match it with the ones in the file.

Reply:
changed